ci-operator/config/openshift-assisted/assisted-installer-deployment/openshift-assisted-assisted-installer-deployment-master.yaml

```diff
@@ -38,6 +38,11 @@ tests:
   container:
     from: assisted-installer-deployment
   postsubmit: true
+- as: unit-test-signatures
```

Comment:
I'd like proposing just ``unit-test`` job, to have less maintenance on the definition of unit tests and more flexibility

Reply:
okay will do